client/src/docs-ui/component-playground/component-playground.styles.scss

```diff
@@ -0,0 +1,4 @@
+amplify-authenticator {
+  --container-height: auto;
+  --container-display: inline;
+}
```

Comment:
This was to account for https://github.com/aws-amplify/amplify-js/pull/6948/files.